editor/import_dock.cpp

```diff
@@ -685,12 +689,25 @@ void ImportDock::_reimport() {
 		}
 
 		config->save(params->paths[i] + ".import");
+
+		//if the list of resources to import includes the currently-edited object in the inspector, save its path so we can load its reimported version later.
```

Comment:
```suggestion
		// If the list of resources to import includes the currently-edited object in the inspector,
		// save its path so we can load its reimported version later.
```

Reply:
I'd remove the hyphen as well:
```suggestion
		// If the list of resources to import includes the currently edited object in the inspector, save its path so we can load its reimported version later.
```